backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -275,15 +275,27 @@ export class UserResolver {
     return response
   }
 
-  @Query(() => CheckUsernameResponse)
-  async checkUsername(
-    @Args() { username, groupId = 1 }: CheckUsernameArgs,
-  ): Promise<CheckUsernameResponse> {
-    const response = await apiGet(
-      CONFIG.LOGIN_API_URL + `checkUsername?username=${username}&group_id=${groupId}`,
-    )
-    if (!response.success) throw new Error(response.data)
-    return new CheckUsernameResponse(response.data)
+  @Query(() => Boolean)
+  async checkUsername(@Args() { username }: CheckUsernameArgs): Promise<boolean> {
+    // Username empty?
+    if (username === '') {
+      throw new Error('Username must be set.')
+    }
+
+    // Do we fullfil the minimum character length?
+    const MIN_CHARACTERS_USERNAME = 2
+    if (username.length < MIN_CHARACTERS_USERNAME) {
+      throw new Error(`Username must be at minimum ${MIN_CHARACTERS_USERNAME} characters long.`)
+    }
+
+    const usersFound = await LoginUser.count({ username })
```

Comment:
Needs to be the LoginUserRepository
```suggestion
    const loginUserRepository = getCustomRepository(LoginUserRepository)
    const usersFound = await loginUserRepository.count({ username })
```